jina/drivers/__init__.py

```diff
@@ -260,6 +260,10 @@ def pre_traverse(_docs, traverse_on, context_doc=None):
         if 'matches' in self.traverse_fields:
             for d in docs:
                 _traverse(d.matches, 'matches')
+        if 'chunk_matches' in self.traverse_fields:
```

Comment:
I think this effect is already achieved by adding chunks, matches. Given that is if, if, first will iterate on chunks and later on matches.

Reply:
@JoanFM is right. no need of this logic . simply set traverse_on to [chunks, matches] does this job.

Reply:
https://github.com/jina-ai/jina/pull/757#issuecomment-667923756

plz see my previous comment.

btw, this PR is really sidetracked and takes too long time. i strongly suggested to close this PR with your added unittest only, if there are fixes on Transformers then started another PR.

or at least change the title of this PR or the issue description #762